packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -524,6 +524,30 @@ export enum RuntimeHeaders {
     viaHandle = "viaHandle",
 }
 
+/** True if a tombstoned object should be returned without erroring */
+export const AllowTombstoneRequestHeaderKey = "allowTombstone"; // Belongs in the enum above, but avoiding the breaking change
+
+/** Tombstone error responses will have this header set to true */
+export const TombstoneResponseHeaderKey = "isTombstoned"
+
+/**
+ * The full set of parsed header data that may be found on Runtime requests
+ */
+export interface RuntimeHeaderData {
+    wait?: boolean;
+    externalRequest?: boolean;
+    viaHandle?: boolean;
+    allowTombstone?: boolean;
+}
```

Comment:
Is this expanding our public api surface?

Reply:
Nope, see index.ts